src/tds/config.c

```diff
@@ -466,10 +466,10 @@ tds_config_boolean(const char *option, const char *value, TDSLOGIN *login)
 static void
 tds_config_encryption(const char * value, TDSLOGIN * login)
 {
-	TDS_ENCRYPTION_LEVEL lvl = TDS_ENCRYPTION_OFF;
+	TDS_ENCRYPTION_LEVEL lvl = TDS_ENCRYPTION_REQUEST;
 
 	if (!strcasecmp(value, TDS_STR_ENCRYPTION_OFF))
-		;
+		lvl = TDS_ENCRYPTION_OFF;
```

Comment:
https://github.com/FreeTDS/freetds, master branch. The specific patch is https://github.com/FreeTDS/freetds/commit/f0400f56a34ce13fd3ece82cdcc21177a0715a8d. I had to change default to off for Sybase and request for MSSQL.
I think the ";" statement was there to avoid some warnings from compilers that don't understand lvl is always set when accessed at the end. The initial lvl initialization is not the default. Note that lvl is set in all paths except where the string value is OFF (in this case lvl is set to OFF).